compiler/src/dmd/nogc.d

```diff
@@ -24,6 +24,7 @@ import dmd.errors;
 import dmd.expression;
 import dmd.func;
 import dmd.globals;
+import dmd.id;
```

Comment:
Why the new import?

Reply:
It was just for debugging and I forgot to remove it. Now it's gone. Thanks for pointing it out.

Reply:
It's still here

Reply:
Gone for good now.